x-pack/test/apm_api_integration/configs/index.ts

```diff
@@ -18,7 +18,7 @@ const apmFtrConfigs = {
   rules: {
     license: 'trial' as const,
     kibanaConfig: {
-      'xpack.ruleRegistry.index': '.kibana-alerts',
```

Comment:
Tracking the ability to abide by changes to the config file, will be fixed in a different PR.

Reply:
Can you clarify why?

Reply:
I think I can actually undo this change..

Reply:
removed here: 48e1b91d7517cacaf5a06fbed3731cdb15cd6de3